### DISCUSSION TITLE
Solo Trip | Jimmy W.P. Lee

### DISCUSSION THREAD

**Discussion by giscus** [alshedivat/al-folio#2704] [Comments]
# Solo Trip | Jimmy W.P. Lee

My legendary solo trip to Japan and Taiwan

https://j4m0fj1mmy.github.io/blog/2023/solo-trip/

**Comment by chrislpy** [comment]
雞雞到此一游

**Comment by chrislpy** [comment]
What a nice trip Jimmy !
